src/components/ErrorBoundary/index.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from "react";
+import styles from "./index.module.css";
+
+class ErrorBoundary extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { hasError: false, error: null, errorInfo: null };
+  }
+
+  componentDidCatch(error, errorInfo) {
+    this.setState({ hasError: true, error: error, errorInfo: errorInfo });
+  }
+
+  render() {
+    if (this.state.hasError) {
+      // Error path
+      return (
+        <div className={styles["boundary"]}>
+          <div className={styles["boundary-wrapper"]}>
+            <h2 className={styles["title"]}>Something went wrong.</h2>
+            <details className={styles["details"]}>
+              {this.state.error && this.state.error.toString()}
+              <br />
+              {this.state.errorInfo && this.state.errorInfo.componentStack}
+            </details>
+          </div>
+        </div>
+      );
+    }
+
+    return this.props.children;
+  }
+}
+export default ErrorBoundary;
+
+// Примечание
```

Comment:
It's a common rule for open source. In the most cases we use english for collaboration. Also for comments.
Maybe somebody from Spain of maybe China or Japan want to use our project. And russian comments make them sad, cause they can't to understand it.

Also comments usually take place above the code they explain, not below. Could you fixed it please for the our colleagues' happiness sake?

Reply:
Fix it for world peace )